OpenRA.Mods.AS/Traits/TeleportNetwork.cs

```diff
@@ -0,0 +1,92 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2015- OpenRA.Mods.AS Developers (see AUTHORS)
+ * This file is a part of a third-party plugin for OpenRA, which is
+ * free software. It is made available to you under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation. For more information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.AS.Traits
+{
+	// TO-DO: Create a proper check for Types of TeleportNetwork and TeleportNetworkManager or lint rule.
+	[Desc("This actor can teleport actors like Nydus canels in SC1. Assuming static object. The actor must have PrimaryBuilding trait so that exit can be specified!")]
```

Comment:
Is PrimaryBuilding even required? it seems like the comment itself is outdated - TNPE does the job.